themes/default/content/docs/clouds/aws/guides/vpc.md

```diff
@@ -64,9 +64,9 @@ VPC is often the easiest path when you're just getting up and running or don't y
 requirements. Most resources will use this default VPC automatically if you leave it unspecified. In other cases,
 you may be required to pass it explicitly, in which case you'll need to get it programmatically.
 
-To get the default VPC, just call the `awsx.vpc.DefaultVpc("default-vpc");` function:
+To get the default VPC, just call the [`awsx.vpc.DefaultVpc` function](https://www.pulumi.com/registry/packages/awsx/api-docs/ec2/defaultvpc/):
```

Comment:
@scottslowe  Given the context, how would you recommend rewording this part? 

I actually removed the ("default-vpc") to help reduce confusion here because the high level name for this function is called `awsx.ec2.DefaultVpc` in the docs, and the code snippets below this shows the language specific way of calling it.

Reply:
Maybe something like "Here are some examples of getting the default VPC across our supported languages"?

Reply:
Updated to the following:

![image](https://github.com/pulumi/pulumi-hugo/assets/19211559/0143a75f-bcee-4ad1-b533-7c461c24f42f)

Reply:
Looks good!